smac/smbo/smbo.py

```diff
@@ -308,3 +309,36 @@ def _get_next_by_local_search(self, init_points=typing.List[Configuration]):
         configs_acq.sort(reverse=True, key=lambda x: x[0])
 
         return configs_acq
+
+    def _get_acq_values_sorted(self, configs):
```

Comment:
I think a more descriptive name of this method would be `_sort_configs_by_acq_value)`.